guide/system-requirements.md

```diff
@@ -19,20 +19,19 @@ Webots on your computer systems to ensure they are compatible before you buy.
 The following operating systems are supported:
 
 - Linux: Webots is ensured to run on the latest Ubuntu Long Term Support (LTS)
-release, currently version 14.04. But it is also known to run on most recent
+release, currently versions 16.04. But it is also known to run on most recent
```

Comment:
Because initially I kept the version 14.04, but then I decided to remove it and I didn't notice that the s remained.